no-computer.md

```diff
@@ -58,12 +67,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Most of the time someone spent on this project would be spent learning the probabilities, designing the conditional tree to determine what conditions should determine each branch. The biggest struggle would be in finding the best balance between detailed complexity which would allow the program to give the player the exact best move every time, and in keeping the program small and simple enough that the average user can keep track of what has been played, correctly answer the conditionals, and perhaps even memorize the conditional tree so that the can run the program in their own mind when they are playing in person with other people.
 
```

Comment:
It might also be interesting to talk to professional cardplayers to see if you can see how much of their intuition is in line with these probabilistic calculations